partner_duplicate_mgmt/init_hook.py

```diff
@@ -1,6 +1,6 @@
 # -*- coding: utf-8 -*-
-# © 2016 Savoir-faire Linux
-# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+# © 2017-2018 Savoir-faire Linux
```

Comment:
2016-2018 plutôt ?

Reply:
@ventilooo no, the module was developped on september 2017